boards/arm/rcar_h3_salvatorx/doc/rcar_h3_salvatorx.rst

```diff
@@ -0,0 +1,188 @@
+.. _rcar_h3_salvatorx_boards:
+
+Renesas R-Car H3 Salvator-X
+###########################
+
+Overview
+********
+- The H3 Salvator-X board is designed for evaluating the features and performance
+  of the R-CAR H3 device from Renesas Electronics and it is also used for developing
+  and evaluating application software for these R-CAR H3.
+
+- The H3 Salvator-X, based on the R-CAR H3 SIP, comes with LPDDR4 @4GB in 2-channel,
+  each 64-bit wide+Hyperflash @64MB, CSI2 interfaces and several communication interfaces
+  like USB, Ethernet, HDMI and can work standalone or can be adapted to other boards,
+  via 440pin connector on bottom side.
+
+.. figure:: img/r_car_h3.jpg
```

Comment:
please add fullname of the board in this file name. `rcar_h3_salvatorx.jpg`

Reply:
Done. Thanks